pkg/proc/native/proc_darwin.c

```diff
@@ -133,67 +133,138 @@ thread_count(task_t task) {
 	return count;
 }
 
+
+const char* exception_string(int code) {
+	if (code == EXC_SOFTWARE) {
+		return "EXC_SOFTWARE";
+	} else if (code == EXC_BREAKPOINT) {
+		return "EXC_BREAKPOINT";
+	} else if (code == EXC_BAD_ACCESS) {
+		return "EXC_BAD_ACCESS";
+	}
+
+	return NULL;
+}
+
+const char* signal_string(int signal) {
+	if (signal == SIGHUP) {
```

Comment:
Maybe like this?

``` c
switch(signal) {
case SIGHUP:
// …
}
```

Reply:
Yeah, I agree, we can switch these if/else chains with a switch.